controllers/articles.controller.js

```diff
@@ -1,4 +1,13 @@
-const {fetchArticleById} = require("../models/articles.model.js");
+const {fetchArticleById, fetchArticles} = require("../models/articles.model.js");
+
+exports.getArticles = (req, res, next) => {
+    const {properties, sortBy, orderBy} = req.query
```

Comment:
As stated in the tests, we shouldn't have a req.query here. The code should by default be sorted in date descending order.